DevelopedCalculator.js

```diff
@@ -30,8 +30,8 @@ class DevelopedCalculator extends Calculator {
     onOperatorButtonClick(event) {
         // calling parent class's onOperatorButtonClick method
         super.onOperatorButtonClick(event);
-        // for adding the clicked operator 
-        this.operator = event.target.dataset.operator;
+        // pushing the history array with current result
```

Comment:
what do you mean by " pushing the history array with current result"

you're pushing the "currentResult" to the history array 
is this correct?